interface/billing/billing_report.php

```diff
@@ -1268,7 +1268,7 @@ function divtoggle(spanid, divid) { //Called in the Expand, Collapse links(This
                             $rhtml .= "</td>\n";
                             $justify = "";
 
-                            if ($iter['id'] && $code_types[$iter['code_type']]['just']) {
+                            if ($iter['id'] && ($code_types[$iter['code_type']]['just'] ?? null)) {
```

Comment:
in this case, would use !empty  (generally works well in all cases where when seeing if a variable is present and "true"(ie. something besides empy/null/0))
```
if ($iter['id'] && !empty($code_types[$iter['code_type']]['just'])) {
```